include/SimCore/Event/SimParticle.h

```diff
@@ -38,6 +38,7 @@ class SimParticle {
     photonNuclear,
     GammaToMuPair,
     eDarkBrem,
+    Decay,
```

Comment:
I am unsure if this will cause an issue with ROOT's serialization - we may see an "hash not equal but same version number" warning if reading events in the old format with the new class (or vice versa); however, since we don't actually store this enum (and just use its `int` cast), ROOT might not complain since we are not actually changing the schema that needs to be serialized. Bascially, I just don't know how ROOT calculates the "hash" of a class and what it decides to serialize.

In any case, even if there is a warning, I think we should go ahead with this update, just want to have a mention of a warning folks might see.

Reply:
I would expect that the serialization would be blind to the names of an enum (especially C-enums that are really just integers)... If it does, I guess we just have to bump the version number?

Reply:
I would also expect the serialization to not actually change, I am just unsure on how ROOT calculates the hash. I _hope_ it only hashes the names and types of the member variables it is saving but it could be doing something like hashing the class declaration (which would include this change) or worse hashing the header file (which would include nonsense like comment changes).

In any case, I'd actually advocate against changing the version number and instead documenting why the warning can be ignored.